dev/performances.js

```diff
@@ -0,0 +1,101 @@
+const puppeteer = require('puppeteer')
+const fs = require('fs')
+const webpack = require('webpack')
+const webpackConfig = require('../build/webpack.config')
+const PORT = 3010
+const HOST_URI = `http://localhost:${PORT}`
+
+
+;(async () => {
+  /* production build */
+  const buildTime = await buildProd()
+
+  /* start host */
+  const appServer = await serverStart()
+
+  /* start puppeteer */
+  const browser = await puppeteer.launch()
+  const page = await browser.newPage()
+
+  /* connect to homepage for performance test */
+  await page.goto(HOST_URI)
+
+  const metrics = await page.metrics()
+  const mapStats = fs.statSync('../public/build/javascript/map.js')
+  const appStats = fs.statSync('../public/build/javascript/bundle.js')
+  await wait(3000)
+  let times = await page.evaluate(() => {
+    return window.times
+  })
+
+  let reportData = {
+    times : {
+      script : metrics.ScriptDuration,
+      build : buildTime,
+      appRender : times.appRendered - times.init,
+      mapLoad : times.mapLoaded - times.init
+    },
+    size: {
+      app : appStats.size,
+      map: mapStats.size
+    }
+  }
+
+  writeReport(reportData)
+  serverClose(appServer)
+  await browser.close()
+})()
+
+function writeReport(reportData) {
+  fs.writeFileSync('./report.json', JSON.stringify(reportData, null, '\t'))
```

Comment:
`__dirname` missing

Reply:
And it would be nice to concatenate the results, instead of rewriting. (With commit and date properties)

Reply:
And printing the current results would be nice too ;)